### PR TITLE
Initial SlackAlerter using tornado_rest_client.clients.slack.

### PR DESCRIPTION
**Pull request by diranged**

### CODE REVIEW

**Comment by diranged** [comment]
@siminm, ready for review.. I squashed your fix in after I improved a few more tests. Thanks for the help though!

File: zk_monitor/alerts/slack.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    def _alert(self, path, state, message, params):
+        log.info("Creating a client from: %s" % slack.Slack)
+        client = slack.Slack(token=params['token'])
+        post_message = client.chat_postMessage()
+
+        icon = self.style_from_state(state)
+
+        status = None
+        try:
+            log.info('About to execute obj: %s' % post_message)
```
Comment (siminm) [inline zk_monitor/alerts/slack.py]: remove this

File: zk_monitor/alerts/slack.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            states.ERROR: ':exclamation:'
+        }
+
+        default = ':grey_question:'
+
+        return styles.get(state) or default
+
+    @gen.coroutine
+    def _alert(self, path, state, message, params):
+        log.info("Creating a client from: %s" % slack.Slack)
```
Comment (siminm) [inline zk_monitor/alerts/slack.py]: remove this

**Comment by siminm** [comment]
:+1: when you remove two log statements

**Pull request merged**